components/automate-gateway/handler/license.go

```diff
@@ -133,7 +133,12 @@ func (t *LicenseServer) RequestLicense(ctx context.Context,
 }
 
 func (t *LicenseServer) getDeploymentID(ctx context.Context) (string, error) {
-	deployIDResponse, err := t.deployClient.DeployID(ctx, &deployment_service.DeployIDRequest{})
+	// deployIDResponse, err := t.deployClient.DeployID(ctx, &deployment_service.DeployIDRequest{})
```

Comment:
Remove the commented code

Reply:
Removed the commented code, please check

Reply:
> The response mentioned in the description does not have the deploymentID. Is it the correct API mentioned?

The given API uses the deployment id from license control service to get the license, did not return it in the response